spec/features/officing_spec.rb

```diff
@@ -169,7 +171,9 @@
       expect(page).to have_content("Here you can validate user documents and store voting results")
 
       visit new_officing_residence_path
-      select 'DNI', from: 'residence_document_type'
+      expect(page).to have_selector('#residence_document_type')
+
+      select 'Passport', from: 'residence_document_type'
```

Comment:
Again.. is the change of "DNI" for "Passport" a mistake or is there a reason behind? If so please explain why.

Reply:
There are none, but come with the cherry pick, I change the commit to the minimum necessary to maintain the functionality in this repository. I had done it thinking in the opposite case, that the repos were similar

Reply:
Madrid is a fork with many particular custom changes, cherry-picking its nice to avoid reproducing changes by hand, but reviewing changes done its always a must for the reviewer, and should be for the author. If human attention wasn't needed we could automate this things entirely and go take some vacations, right?

Reply:
True, I take note